autoscaling/autoscaling.go

```diff
@@ -298,6 +355,7 @@ func (autoscaling *AutoScaling) CreateLaunchConfiguration(options *CreateLaunchC
 		b64.Encode(userData, []byte(options.UserData))
 		params["UserData"] = string(userData)
 	}
+	addBlockDeviceParams("", params, options.BlockDevices)
```

Comment:
I'm guessing he just copied the function from another package. Its fine, we'll switch to aws-go soon enough anyways.